pkgs/top-level/all-packages.nix

```diff
@@ -22758,6 +22758,8 @@ in
 
   synergyWithoutGUI = synergy.override { withGUI = false; };
 
+  pipr = callPackage ../applications/misc/pipr { };
```

Comment:
The file is somehwat sorted alphabetically, can you please try and find a better spot to put this?